app/default_overrides.go

```diff
@@ -127,3 +130,24 @@ func getLegacyProposalHandlers() (result []govclient.ProposalHandler) {
 
 	return result
 }
+
+// DefaultConsensusParams returns a ConsensusParams with a MaxBytes
+// determined using a goal square size.
+func DefaultConsensusParams() *tmproto.ConsensusParams {
+	return &tmproto.ConsensusParams{
+		Block:     DefaultBlockParams(),
+		Evidence:  coretypes.DefaultEvidenceParams(),
+		Validator: coretypes.DefaultValidatorParams(),
+		Version:   coretypes.DefaultVersionParams(), // TODO: set the default version to 1
+	}
+}
+
+// DefaultBlockParams returns a default BlockParams with a MaxBytes determined
+// using a goal square size.
+func DefaultBlockParams() tmproto.BlockParams {
+	return tmproto.BlockParams{
+		MaxBytes:   appconsts.DefaultMaxBytes,
+		MaxGas:     -1,
+		TimeIotaMs: 1000, // 1s
```

Comment:
Why not 1ms?

It has been hardcoded to 1ms in mainline comet (See https://github.com/cometbft/cometbft/blob/748fe9501226dd3b7a86c514fb668096fe81a1cb/consensus/state.go#L2346)

Reply:
this is just what it used to be, but we can definitely use 1ms. c5c09b3a23e483648c5cbc03501817fbcc03a66e

question tho, if its 1ms, shouldn't we (royal we as in everyone who uses comet) just get rid of it?